src/pages/teachers/programmes/[programmeSlug]/units.tsx

```diff
@@ -98,16 +99,27 @@ const UnitListingPage: NextPage<UnitListingPageProps> = ({
     ...{ noFollow: true, noIndex: true },
   };
 
-  const trackUnitSelected = ({ ...props }: UnitListItemProps) => {
-    track.unitSelected({
-      keyStageTitle: props.keyStageTitle as KeyStageTitleValueType,
-      keyStageSlug,
-      subjectTitle,
-      subjectSlug,
-      unitName: props.title,
-      unitSlug: props.slug,
-      analyticsUseCase,
-    });
+  const trackUnitSelected = ({
+    ...props
+  }: UnitListItemProps | IndividualSpecialistUnit) => {
+    // Temporary until tracking for specialist units
+    const isSpecialistUnit = (
+      x: UnitListItemProps | IndividualSpecialistUnit,
+    ): x is IndividualSpecialistUnit => {
+      return "developmentalStageTitle" in x;
+    };
+
+    if (!isSpecialistUnit(props)) {
```

Comment:
specialist pages will be in a different folder/route (`specialistProgrammes/[programmeSlug]/specialistUnits`) so this page shouldn't need to worry about there being any specialist data